.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,12 +9,14 @@ For more information please refer to our [contributing documentation][contributi
 ## I Am Adding a New Code Snippet in an Existing Language
 
 - [ ] I fixed #your-issue-number-here
+- [ ] I removed imported frameworks/libraries/packages or added them to .gitignore
```

Comment:
I would probably write this as “I did not include any extra folders or libraries” or something similar. I think remove might confuse some folks into thinking they need to remove something.